src/components/File/MilkdownEditor.tsx

```diff
@@ -27,7 +27,7 @@ const MilkdownEditor: React.FC<MarkdownEditorProps> = ({
 
   const saveFile = async () => {
     if (content !== lastSavedContentRef.current) {
-      await window.files.writeFile(filePath, content);
+      await window.files.writeFile(filePath, content, false);
```

Comment:
Otherwise now u are relying on having to know the structure, and writing in the values with the correct propName for readability

In this case, `false` isn't super clear to read, unless u already know that its referring to whether to index or not